pilot/pkg/config/memory/store.go

```diff
@@ -272,11 +251,12 @@ func (cr *store) Patch(orig config.Config, patchFn config.PatchFunc) (string, er
 		return "", errNotFound
 	}
 
-	rev := time.Now().String()
-	cfg.ResourceVersion = rev
+	if cfg.ResourceVersion == "" {
+		cfg.ResourceVersion = time.Now().String()
```

Comment:
shouldn't patch update the resource version?

Reply:
> shouldn't patch update the resource version?

Should be updated